moto/sqs/responses.py

```diff
@@ -246,6 +248,7 @@ def send_message(self):
             delay_seconds=delay_seconds,
             deduplication_id=message_dedupe_id,
             group_id=message_group_id,
+            aws_trace_header=aws_trace_header,
```

Comment:
With the above change, we can just store the `system_attributes` as a whole. The exact properties are not relevant to this method. 
For now it may only contain the aws_trace_header, but if AWS ever decides to store more attributes, we don't have to change anything

Reply:
Done,Changed as requested